beater/api/config/agent/handler.go

```diff
@@ -187,6 +187,12 @@ func extractInternalError(c *request.Context, err error, withAuth bool) {
 		body = authErrMsg(msg, agentcfg.ErrMsgReadKibanaResponse, withAuth)
 		keyword = agentcfg.ErrMsgReadKibanaResponse
 
+	case strings.Contains(msg, agentcfg.ErrUnauthorized):
+		fullMsg := "APM Server is not authorized to contact Kibana. " +
+			"Please configure apm-server.kibana.username and apm-server.kibana.password"
```

Comment:
Would this error occur if the server had username/password configured, but the username didn't have the right privileges? If so, perhaps reword a bit, like:

"APM Server is not authorized to query agent configuration via Kibana. Please configure apm-server.kibana.username and apm-server.kibana.password, and ensure the user has the necessary privileges."